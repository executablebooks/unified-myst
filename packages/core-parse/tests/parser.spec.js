import { Processor } from '../src/index.js'

describe('Parser', () => {
    test('parse', () => {
        const parser = new Processor()
        const result = parser.toAst('hallo world!')
        expect(result).toMatchSnapshot()
    })
})
