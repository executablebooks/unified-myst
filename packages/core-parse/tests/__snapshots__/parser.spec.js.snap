// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parser parse 1`] = `
Object {
  "ast": Object {
    "children": Array [
      Object {
        "children": Array [
          Object {
            "position": Object {
              "end": Object {
                "column": 13,
                "line": 1,
                "offset": 12,
              },
              "start": Object {
                "column": 1,
                "line": 1,
                "offset": 0,
              },
            },
            "type": "text",
            "value": "hallo world!",
          },
        ],
        "position": Object {
          "end": Object {
            "column": 13,
            "line": 1,
            "offset": 12,
          },
          "start": Object {
            "column": 1,
            "line": 1,
            "offset": 0,
          },
        },
        "type": "paragraph",
      },
    ],
    "position": Object {
      "end": Object {
        "column": 13,
        "line": 1,
        "offset": 12,
      },
      "start": Object {
        "column": 1,
        "line": 1,
        "offset": 0,
      },
    },
    "type": "root",
  },
  "state": Object {},
}
`;
